lighthouse-core/scripts/lantern/assert-master-lantern-values-unchanged.js

```diff
@@ -10,50 +10,62 @@
 
 const fs = require('fs');
 const path = require('path');
-const execSync = require('child_process').execSync;
 const constants = require('./constants');
 
+// @ts-ignore
+const chalk = require('chalk');
+
 const INPUT_PATH = process.argv[2] || constants.SITE_INDEX_WITH_GOLDEN_WITH_COMPUTED_PATH;
 const HEAD_PATH = path.resolve(process.cwd(), INPUT_PATH);
 const MASTER_PATH = constants.MASTER_COMPUTED_PATH;
 
-const TMP_DIR = path.join(__dirname, '../../../.tmp');
-const TMP_HEAD_PATH = path.join(TMP_DIR, 'HEAD-for-diff.json');
-const TMP_MASTER_PATH = path.join(TMP_DIR, 'master-for-diff.json');
-
-if (!fs.existsSync(TMP_DIR)) fs.mkdirSync(TMP_DIR);
-
 if (!fs.existsSync(HEAD_PATH) || !fs.existsSync(MASTER_PATH)) {
   throw new Error('Usage $0 <computed file>');
 }
 
-let exitCode = 0;
-
-try {
-  const computedResults = require(HEAD_PATH);
-  const expectedResults = require(MASTER_PATH);
-
-  const sites = [];
-  for (const entry of computedResults.sites) {
-    const lanternValues = entry.lantern;
-    Object.keys(lanternValues).forEach(key => lanternValues[key] = Math.round(lanternValues[key]));
-    sites.push({url: entry.url, ...lanternValues});
-  }
-
-  fs.writeFileSync(TMP_HEAD_PATH, JSON.stringify({sites}, null, 2));
-  fs.writeFileSync(TMP_MASTER_PATH, JSON.stringify(expectedResults, null, 2));
-
-  try {
-    execSync(`git --no-pager diff --color=always --no-index ${TMP_MASTER_PATH} ${TMP_HEAD_PATH}`);
-    console.log('✅  PASS    No changes between expected and computed!');
-  } catch (err) {
-    console.log('❌  FAIL    Changes between expected and computed!\n');
-    console.log(err.stdout.toString());
-    exitCode = 1;
-  }
-} finally {
-  fs.unlinkSync(TMP_HEAD_PATH);
-  fs.unlinkSync(TMP_MASTER_PATH);
+const computedResults = require(HEAD_PATH);
+const expectedResults = require(MASTER_PATH);
```

Comment:
since master is constant, would it be better to require it directly so the compiler can pick it up?

Reply:
I don't think so unless I'm misunderstanding. TS just seems to put on a compiler error on the JSON for not being a module and then we have to copy/paste the path around...?